multi-agent/adapters/inbound/flask/endpoints/collaboration/presence.py

```
@@ -0,0 +1,174 @@
+        participants = svc.join_session(
+            session_id=session_id,
+            user_id=user_id,
+            display_name=display_name,
+            role=participant_role,
+        )
+        return jsonify(participants.model_dump(mode="json")), 200
+    except KeyError:
+        return jsonify({"error": f"Session {session_id} not found"}), 404
+    except ValueError as e:
+        return jsonify({"error": str(e)}), 400
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)